### ISSUE TITLE
Approve Content State 1.0

### ISSUE THREAD

**Issue by glenrobson** [IIIF/trc#86]
## Links

 * Candidate specification: https://iiif.io/api/content-state/0.9/
 * Non-technical summary: https://tom-crane.medium.com/what-is-iiif-content-state-dd15a543939f
 * Encoding demonstrator: https://base64url.herokuapp.com/

 * Pull Request: https://github.com/IIIF/api/pull/2104
 * Preview: https://preview.iiif.io/api/content_state_10/content-state/1.0/

## Background and Summary

The Discovery TSG believe that the Content State API is now ready to be approved as version 1.0. A previous version (0.9) was taken to TRC in issue #79 but withdrawn due to issues raised in the TRC call. We have hopefully addressed the concerns with the following changes:

 * Encoding is now only required in the [GET request parameter](https://preview.iiif.io/api/content_state_10/content-state/1.0/#initialization-mechanisms-link) 
 * The encoding discussion has been moved to the lower part of the specification [Section 6](https://preview.iiif.io/api/content_state_10/content-state/1.0/#6-content-state-encoding) to make the specification easier to read. 
 * The suggested method for encoding was also switched to `encodeURIComponent` in response to the [TRC comment](https://github.com/IIIF/trc/issues/79#issuecomment-951104819).

We have seen the following implementations of content state:

Getty
 - Animal Crossing Art Generator - https://experiments.getty.edu/ac-art-generator/
 - https://github.com/workergnome/content-state#readme
 
University of Durham 
 - https://www.durhampriory.ac.uk/using-the-features-of-iiif/

Biblissima
 - https://twitter.com/regisrob/status/1328398730207387648?s=20

Digirati
 - Consumer and publisher as part of Muya (Multimedia Yasna project, in progress project for SOAS)
 - A related IIIF Discovery platform with cross collection/institution search;
 - Publisher as part of Madoc (Crowdsourcing environment)

## Proposed Solution

The [Discovery TSG](https://iiif.io/community/groups/discovery/) would like the TRC to support the move to a full 1.0 release of the Content State API.

**Comment by thehabes** [comment]
`The target could be any resource described by the Presentation API, for example, a:`

Link directly to the defined types.  Don't end with ", for example, a:"

`The target could be any resource [described by the Presentation API](https://iiif.io/api/presentation/3.0/#21-defined-types) such as a:`

**Comment by thehabes** [comment]
"A content state Annotation must have the motivation contentState. This motivation is not defined by either the [W3C Web Annotation Data Model](http://w3.org/TR/annotation-model/) or the IIIF Presentation API..."

Then where is `contentState` defined?  Do you intend that the vocabulary for that term be in this document, and that the Presentation API 3 will contain the term and link like it does for `iiif_prezi: supplementing` or `iiif_prezi:painting`?

**Comment by thehabes** [comment]
"The target of the annotation is, in this case, a complete IIIF resource (here, a Manifest) but in more complex cases, the target could be a part of a IIIF resource."

Capitalize 'annotation'

After writing I found a couple more.  Check all the instances of "annotation", especially "Content State annotation" vs "Content State Annotation".

**Comment by thehabes** [comment]
"The content state may be supplied as a string whose value is the id (the dereferenceable URI) of the target property only. This is the simplest form and is just the URI of a resource. For the example in 2.2.1, this would be the URI https://example.org/iiif/item1/manifest."

Why not actually add the code block showing `"target":"https://example.org/iiif/item1/manifest"`?

**Comment by thehabes** [comment]
Capitalization around "IIIF Resource" is inconsistent (I think just the one in `4.` is capitalized).

**Comment by thehabes** [comment]
Section 3 is spectacular, I love that you showed the "invalid" way of providing the resources.  This section really made the entire thing make sense, well done.

**Comment by thehabes** [comment]
As a note, many TRC members expressed the need to see Cookbook recipe entries with implementations of content state.  This will help us decide the narrative of error handling and best practices.  No author is on the hook yet.  If this is the deciding factor between a +1/-1 for you, the Cookbook accepts all authors...

**Comment by thehabes** [comment]
As a note, multiple +1s on the idea of a "viewer support matrix" like the[ one available for the Cookbook](https://iiif.io/api/cookbook/recipe/matrix/
).

**Comment by azaroth42** [comment]
> Then where is `contentState` defined? Do you intend that the vocabulary for that term be in this document 

Yes :) Just as Presentation defines `painting` and `supplementing`.

**Comment by glenrobson** [comment]
### Issue 86 (Approve Content State 1.0)
  +1: 23 [azaroth42 cubap dlpierce glenrobson hadro irv jpadfield jtweed julsraemy kirschbombe ksclarke markpatton markpbaggett mbennett-uoe mixterj mposton-folger nfreire regisrob robcast shuddles tgra thehabes zimeon]
   0: 0 []
  -1: 0 []
  Not TRC: 1 [stephenwf]
  Ineligible: 4 [akrishnan15 cjnishioka mattmcgrattan tomcrane]

### Result: 23 / 23 = 1.00
Super majority is in favor, issue is approved
